tests/e2e/default/contacts/edit.wdio-spec.js

```diff
@@ -0,0 +1,30 @@
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const userFactory = require('../../../factories/cht/users/users');
+const contactPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
+const utils = require('../../../utils');
+
+
+describe('Editing contacts with the default config. ', () => {
```

Comment:
The name of the test is `Editing contacts...` but checking the `it` i don't see that the contact name or age or note are edited. 
Should we consider changing the name of the test to be something like `Remove primary contact`? or maybe add another `it` that edit some information about the contact and verify that it was saved successfully?

Reply:
Great idea @tatilepizs! I will complete this spec with another `it` with the actual edition of a contact. Thanks!